test/metabase/models/setting_test.clj

```diff
@@ -581,6 +581,68 @@
            (toucan-name)))))
 
 
+;;; ------------------------------------------------- Setting Visibility ------------------------------------------------
+
+#_{:clj-kondo/ignore [:unused-private-var]}
+(defsetting ^:private test-internal-setting
```

Comment:
Settings are registered globally anyway so marking them private doesn't really matter. Might be better to remove that and then not have to worry about clj-kondo

Reply:
Done